test/tools/yulInterpreter/EVMInstructionInterpreter.cpp

```diff
@@ -314,6 +314,11 @@ u256 EVMInstructionInterpreter::eval(
 		accessMemory(arg[0], arg[1]);
 		logTrace(_instruction, arg);
 		return 0;
+	case Instruction::TLOAD:
+		return m_state.storage[h256(arg[0])];
```

Comment:
This will need as separate key-value store in ``m_state``, i.e. you'll need to add
``std::map<util::h256, util::h256> triansientStorage;`` to ``InterpreterState``. Also a quick look at whether we do anything specially for ``InterpreterState::storage`` would be good (e.g. if we reset it somewhere). Eventually for actual fuzzing/tracing, we will need an analog of ``InterpreterState::dumpStorage(std::ostream& _out) const;`` as well, but we can add that in a separate PR with fuzzing support for transient storage (@bshastry may want to get to that eventually - in that PR we can also check if we need to do anything further to account for the special lifetime of transient storage in the fuzzing setup)